src/platform/intel/ace/platform.c

```diff
@@ -80,18 +81,22 @@ static struct pm_notifier pm_state_notifier = {
 	.state_exit = cpu_notify_state_exit,
 };
 
+#if CONFIG_KCPS_DYNAMIC_CLOCK_CONTROL
+/* Value to be determined experimentaly */
+#define BASE_CPS_USAGE 10000
+#else
+#define BASE_CPS_USAGE (CLK_MAX_CPU_HZ / 1000)
+#endif
```

Comment:
Would this be platform specific or does it make sense to use Kconfig here ?

Reply:
CLK_MAX_CPU_HZ this should be platform specific

Reply:
it could be platform specific but probably we can move this to more appropriate place